src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles.css/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__image"
+      >
+      <p class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__code">
+        Product code: 195434
+      </p>
```

Comment:
```suggestion
      <p class="card__name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </p>
      
      <p class="card__code">
        Product code: 195434
      </p>
```
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) p.3 check it all over your code

Reply:
done